ansible/roles/miniflux/defaults/main.yml

```diff
@@ -155,7 +147,7 @@ miniflux__upstream_type: 'apt'
 # .. envvar:: miniflux__upstream_version [[[
 #
 # Version of the Miniflux upstream.
-miniflux__upstream_version: '2.0.35'
+miniflux__upstream_version: '2.0.48'
```

Comment:
It can be bumped to [2.2.1](https://miniflux.app/releases/2.2.1.html) which is what I use it